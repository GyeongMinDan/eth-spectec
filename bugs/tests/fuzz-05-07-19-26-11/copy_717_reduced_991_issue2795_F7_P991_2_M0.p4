// Intended pid 991
// Source vid 80
// Depth 4

// Mutation MixopGroup

/*
From (ExprA (SeqE [ (NumE (INT +0)),
  (NumE (INT +1)) ]))
To (ExprA (SeqDefaultE [ (NumE (INT +0)),
  (NumE (INT +1)) ]))
*/

extern packet_out {
  void emit<T>(in T hdr);
}
header H {
  bit<32> a;
  bit<32> b;

}
control c(packet_out p)() {

  apply {
    p.emit(((H) ({ 0, 1 })));
    p.emit<H>({ 0, 1, ... });
  }
}

// Covered pids { 717 }
