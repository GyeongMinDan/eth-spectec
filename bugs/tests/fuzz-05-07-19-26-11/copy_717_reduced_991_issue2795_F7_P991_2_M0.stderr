hits-likely-05-07-19-26-11/copy_717_reduced_991_issue2795_F7_P991_2_M0.p4(22): [--Wwarn=unused] warning: Control c is not used; removing
control c(packet_out p)() {
        ^
[--Wwarn=missing] warning: Program does not contain a `main' module
